package.json

```diff
@@ -7,6 +7,8 @@
     "aws-sdk": "^2.139.0",
     "aws-serverless-express": "^3.0.2",
     "body-parser": "^1.18.2",
+    "builder-pattern": "^1.0.1",
+    "cancan": "https://github.com/Vincent-Pang/cancan.git#develop",
```

Comment:
Cancan is a library for authorization, the owner has not merge my pull request yet. So I yarn add from my github first.